src/utils/collisions.ts

```diff
@@ -0,0 +1,44 @@
+import type { Collider, ColliderHandle, RigidBody, World } from '@dimforge/rapier3d-compat'
+import type { Group, Scene } from 'three'
+import type { Ref } from 'vue'
+
+interface CollisionSource {
+  collider: Collider
+  rigidBody: RigidBody | undefined
+};
+
+interface sourceTarget {
+  group: Group
+  rapierGroup: CollisionSource
+}
```

Comment:
To keep the architecture clean, we can create a new file under `src/types/` and call it `collision.ts` and move those two interfaces there